src/dfx/src/lib/models/canister.rs

```diff
@@ -792,7 +792,7 @@ impl CanisterPool {
                     .map(|c| c.get_name())
                     .contains(&canister.get_name())
                 {
-                    trace!(log, "Building canister '{}'.", canister.get_name());
+                    info!(log, "Building canister '{}'.", canister.get_name());
```

Comment:
```suggestion
                    debug!(log, "Building canister '{}'.", canister.get_name());
```
This is already visible with a spinner. I don't think we should duplicate this message unless we're running with `--verbose`

Reply:
> This is already visible with a spinner. I don't think we should duplicate this message unless we're running with `--verbose`

The spinner does show something but finally clears the output.